components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -93,28 +80,62 @@ - (void)setSelectedItem:(UITabBarItem *)selectedItem {
   _selectedItem = selectedItem;
 }
 
-#pragma mark - UIView
+- (void)setUpSubviews {
+  self.backgroundColor = UIColor.whiteColor;
 
-// TODO(#7645): Temporary layout until
-// https://github.com/material-components/material-components-ios/issues/7645 lands
-- (void)layoutSubviews {
-  [super layoutSubviews];
+  [self setUpStackView];
+  [self setUpItemViews];
 
-  if (self.items.count == 0) {
-    return;
+  [NSLayoutConstraint activateConstraints:@[
+    [_stackView.leadingAnchor constraintEqualToAnchor:self.leadingAnchor],
+    [_stackView.trailingAnchor constraintEqualToAnchor:self.trailingAnchor],
+    [_stackView.widthAnchor constraintGreaterThanOrEqualToAnchor:self.widthAnchor],
+    [_stackView.topAnchor constraintEqualToAnchor:self.topAnchor],
+    [_stackView.bottomAnchor constraintEqualToAnchor:self.bottomAnchor],
+  ]];
+}
+
+- (void)setUpStackView {
+  _stackView = [[UIStackView alloc] init];
+  _stackView.axis = UILayoutConstraintAxisHorizontal;
+  _stackView.translatesAutoresizingMaskIntoConstraints = NO;
+  [self addSubview:_stackView];
+}
+
+- (void)setUpItemViews {
+  for (UIView *view in self.stackView.arrangedSubviews) {
+    [view removeFromSuperview];
+    [_stackView removeArrangedSubview:view];
+  }
+
+  for (UITabBarItem *item in self.items) {
+    MDCTabBarViewItemView *itemView = [[MDCTabBarViewItemView alloc] init];
+    itemView.translatesAutoresizingMaskIntoConstraints = NO;
+    itemView.title = item.title;
+    itemView.image = item.image;
```

Comment:
I have removed the title and the image in another PR. Please wait for that to be merged and then update this.

Reply:
Done.